ostd/src/arch/x86/trap/mod.rs

```diff
@@ -220,29 +221,58 @@ pub fn is_kernel_interrupted() -> bool {
 /// Handle traps (only from kernel).
 #[no_mangle]
 extern "sysv64" fn trap_handler(f: &mut TrapFrame) {
-    match CpuException::to_cpu_exception(f.trap_num as u16) {
+    let cpu_exception = CpuException::to_cpu_exception(f.trap_num as u16);
```

Comment:
Move this onto the `match` statement: `match CpuException::to_cpu_exception(f.trap_num as u16)`